harvester_e2e_tests/integrations/test_1_images.py

```diff
@@ -318,6 +318,9 @@ def test_delete_image_recreate(
         get_image(api_client, unique_name)
         delete_image(api_client, unique_name, wait_timeout)
 
+        get_image(api_client, image_name)
```

Comment:
The file of `image_name` should not existed as it already deleted in previous line?
Could you double confirm that it is caused by this test case or others?
(or do you have the test report for reproduce?)

Reply:
Thanks for the comment, what you mentioned in [L319](https://github.com/harvester/tests/blob/4d4d22b397e4fa8ff73ce193c6fce6b78624d356/harvester_e2e_tests/integrations/test_1_images.py#L319) is to delete image name of `unique_name`, while L322 (this pr) wants to delete image name of `image_name` which was created in [L300](https://github.com/harvester/tests/blob/4d4d22b397e4fa8ff73ce193c6fce6b78624d356/harvester_e2e_tests/integrations/test_1_images.py#L300).

Below is the video of testing `delete_image_recreate` harvester UI image page (before patching this pr) by using command as follows:
```sh
tox -e python3.11 --result-json=test_result.json -- -l harvester_e2e_tests --json-report --html=test_report/test_result.html -m 'not delete_host and not upgrade' -k 'delete_image_recreate or create_image_url'
```

https://github.com/user-attachments/assets/79c1b3fc-63da-42c7-9d84-3d9721bf01d7

As you can see, there were two images remained after test finished, `opensuse-11h10m13s466515-09-20` and `ubuntu-11h10m13s466515-09-20`.

Reply:
@lanfon72 If agreed with @brandboat's comment, please resolve the conversation and proceed.

Reply:
No, it is actually a test case design problem.

The image `image_name` which created in test case **test_create_image_url**, and it will be used in **test_delete_image_recreate** and **test_edit_image_in_use**, and finally be deleted in **test_edit_image_in_use** (the last line)

so in this case, if we just execute those two test cases, the image will exist is expected. (by the implementation)

and if we include this change, the test case **test_edit_image_in_use** will fail in the first assertion.